javascript/nanoexpress/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress": "^1.1"
```

Comment:
rather use tilde @dalisoft 

to stick into the minor

use carret could allow usage of `1.2` or `1.9`

Reply:
I think @dependabot does the job for update, right?

I changed :)

Reply:
Yes